### PR TITLE
Archiving and deleting Pipelines

### PR DESCRIPTION
**Pull request by JuanitoFatas**
https://docs-review-docs-pipeli-belw8w.herokuapp.com/docs/pipelines/managing-pipelines

Screen Shot 2020-12-15 at 12 46 38

Will be https://buildkite.com/docs/pipelines/managing-pipelines after shipping this.

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
@plaindocs Love a review from you!

File: pages/pipelines/managing_pipelines.md.erb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Managing Pipelines
+
+{:toc}
+
+## Archiving Pipeline
+
+You can archive/unarchive a pipeline if you’re an administrator of organization or in a team that has Full Access to the pipeline.
```
Comment (plaindocs) [inline pages/pipelines/managing_pipelines.md.erb:7]:
```suggestion
You can archive/unarchive a pipeline if you’re an administrator of the Buildkite organization or in a team that has Full Access to the pipeline.
```

File: pages/pipelines/managing_pipelines.md.erb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Archiving a pipeline will preserve all builds, job logs, artifacts and history for the pipeline. Archived Pipelines are hidden on the Pipelines page and new builds won’t run.
```
Comment (plaindocs) [inline pages/pipelines/managing_pipelines.md.erb:9]:
```suggestion
Archiving a pipeline preserves all builds, job logs, artifacts and history for the pipeline. Archived Pipelines are hidden on the Pipelines page and won’t run new builds.
```

File: pages/pipelines/managing_pipelines.md.erb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+1. Navigate to Pipeline page
+2. click Pipeline Settings
+3. Under "Pipeline Management", click "Archive Pipeline"/"Unarchive Pipeline"
+4. Read the warnings
```
Comment (plaindocs) [inline pages/pipelines/managing_pipelines.md.erb:14]: <3

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -201,6 +201,7 @@
                   <li class="Docs__nav__sub-nav__item"><%= sidebar_link_to "Notifications", 'pipelines/notifications' %></li>
                   <li class="Docs__nav__sub-nav__item"><%= sidebar_link_to "Emojis", 'pipelines/emojis' %></li>
                   <li class="Docs__nav__sub-nav__item"><%= sidebar_link_to "Example Pipelines", 'pipelines/example-pipelines' %></li>
+                  <li class="Docs__nav__sub-nav__item"><%= sidebar_link_to "Managing Pipelines", 'pipelines/managing-pipelines' %></li>
```
Comment (plaindocs) [inline app/views/layouts/application.html.erb:204]: What do you think about "Archiving and deleting PIpelines" here? Managing makes me think of organizing them.

**Review by plaindocs — APPROVED** [review]
This is great, some suggestions inline.

**Comment by plaindocs** [comment]
Awesome!

**Pull request merged**